driver-service/src/main/java/com/modsen/driverservice/service/impl/DriverServiceImpl.java

```diff
@@ -26,6 +27,7 @@
 @Service
 @RequiredArgsConstructor
 @Slf4j
+@Transactional
```

Comment:
It's safer now, but this approach puts additional performance costs.
Here's an [interesting article about read only transactions ]( https://medium.com/@jkha7371/is-transactional-readonly-true-a-silver-bullet-1dbf130c97f8)
Please adjust your transactions properly